src/histolab/filters/image_filters.py

```diff
@@ -715,14 +722,30 @@ class RagThreshold(ImageFilter):
     """  # noqa
 
     def __init__(
-        self, n_segments: int = 800, compactness: float = 10.0, threshold: int = 9
+        self,
+        n_segments: int = 800,
+        compactness: float = 10.0,
+        threshold: int = 9,
+        return_labels: bool = False,
     ) -> PIL.Image.Image:
         self.n_segments = n_segments
         self.compactness = compactness
         self.threshold = threshold
+        self.return_labels = return_labels
 
-    def __call__(self, img: PIL.Image.Image) -> PIL.Image.Image:
-        return F.rag_threshold(img, self.n_segments, self.compactness, self.threshold)
+    def __call__(
+        self,
+        img: PIL.Image.Image,
+        mask: np.ndarray = None,
+    ) -> PIL.Image.Image:
```

Comment:
if we agree with the current implementation, this is not `True` anymore. We should have `Union[ndarray, PIL.Image]`

Reply:
yeah I put it on my suggestions... we were reviewing at the same time